css/at-rules.json

```
@@ -455,7 +455,7 @@
         "status": "experimental"
       },
       "initial-value": {
-        "syntax": "<string>",
+        "syntax": "<declaration-value>",
```

```suggestion
        "syntax": "<declaration-value>?",
```
https://drafts.css-houdini.org/css-properties-values-api/#initial-value-descriptor